catalyst/dl/core/runner.py

```diff
@@ -197,5 +197,9 @@ def run_experiment(self, experiment: Experiment, check: bool = False):
 
         return self
 
+    @abstractmethod
+    def forward(self, batch: Mapping[str, Any]) -> Mapping[str, Any]:
```

Comment:
why do we need it? we already have predict_batch for this purpose.
I think it is better to leave for later PR